README.md

```diff
@@ -330,6 +332,81 @@ extraObjects:
               restartPolicy: OnFailure
 ```
 
+## git-sync
+
+`extraObjects`, `extraContainers`, `extraInitContainers`, `extraVolumes`, and `extraVolumeMounts` can be combined to deploy git-sync. The following example relies on `emptyDir` volumes and works with `KubernetesExecutor`.
+
+```yaml
+env:
+  - name: AIRFLOW__CORE__DAGS_FOLDER
+    value: /usr/local/airflow/dags/dags/airflow/example_dags
```

Comment:
Is the path correct? this specifically `airflow/dags/dags/airflow`?

Reply:
Yeah, exactly. I could choose another DEST so it is more clear.